lib/fluent/plugin/query_params_extractor.rb

```diff
@@ -0,0 +1,132 @@
+require 'uri'
+require 'cgi/util'
+require 'webrick'
```

Comment:
Yes, we are using followings for now:

* `URI.parse`
* `WEBrick::HTTPUtils.escape`
* `CGI.unescape`